src/styles/annotator/inject.scss

```diff
@@ -9,6 +9,7 @@ $base-font-size: 14px;
 .annotator-highlights-always-on {
   .annotator-hl {
     background-color: $highlight-color;
```

Comment:
Might be useful to extract this `background-color: ...; fill: ...;` property pair which are always used together into a mixin and add a brief note about the reason for having both properties.